### PR TITLE
Increase SERVER_TIMEOUT for L0_infer_valgrind

### PR DESCRIPTION
**Pull request by krishung5**
Related PRs:
common: https://github.com/triton-inference-server/common/pull/67
backend: https://github.com/triton-inference-server/backend/pull/67
tensorrt_backend: https://github.com/triton-inference-server/tensorrt_backend/pull/44

### CODE REVIEW

**Review by tanmayv25 — DISMISSED** [review]

**Comment by GuanLuo** [comment]
It shouldn't take this long (2 days) to load models, right? When did the test start timing out?

**Comment by krishung5** [comment]
@GuanLuo I don't expect the test to takes 2 days long either. I set it to 2 days just in case it runs over the TIMEOUT and we need to restart it all over again. In the last run, the test got killed because there is a 8h timeout set in our CI pipeline configuration.

**Comment by krishung5** [comment]
Removed onnx and python backends for valgrind tests. Added a possible memory leak introduced from [OpenVINO side](https://github.com/openvinotoolkit/openvino/issues/6593). Made some changes in [OpenVINO backend](https://github.com/triton-inference-server/openvino_backend/pull/40).

**Comment by krishung5** [comment]
Rebased

File: qa/L0_infer/test.sh [hunk 1]
```diff
@@ -63,6 +63,9 @@ if [ "$TEST_VALGRIND" -eq 1 ]; then
     LEAKCHECK_ARGS_BASE="--leak-check=full --show-leak-kinds=definite --max-threads=3000 --num-callers=20"
     SERVER_TIMEOUT=4000
     rm -f $LEAKCHECK_LOG_BASE*
+    # Remove onnx and python backends for now as the server hangs up when
```
Comment (tanmayv25) [inline qa/L0_infer/test.sh:66]: Did you create follow-up tickets to investigate this? If yes, then can you add the DLIS-<IDs> here?

Comment (krishung5) [reply qa/L0_infer/test.sh:66]: Added ticket number.

**Comment by krishung5** [comment]
Although L0_infer_valgrind passes locally, it seems like running `test_class_bbb` test case with TF models on CI would fail due to timeout issue. Increased network_timeout for this case.

**Comment by krishung5** [comment]
L0_infer_valgrind passes on CI: https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/jobs/42488617

File: qa/L0_infer/infer_test.py [hunk 2]
```diff
@@ -56,8 +57,14 @@
...
-    def _full_exact(self, input_dtype, output0_dtype, output1_dtype,
-                    output0_raw, output1_raw, swap):
+    def _full_exact(self,
+                    input_dtype,
+                    output0_dtype,
+                    output1_dtype,
+                    output0_raw,
+                    output1_raw,
+                    swap,
+                    network_timeout=60.0):
```
Comment (tanmayv25) [inline qa/L0_infer/infer_test.py:67]: Why are we adding the network timeout for this test?

Comment (krishung5) [reply qa/L0_infer/infer_test.py:68]: We only increase `network_timeout `for TensorFlow models for valgrind test. The default value of  `network_timeout ` is 60.0. (https://github.com/triton-inference-server/client/blob/main/src/python/library/tritonclient/http/__init__.py#L155)

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**